lib/pages/filter/view/relevance_picker.dart

```diff
@@ -105,7 +105,9 @@ class _RelevancePickerState extends State<RelevancePicker> {
                 child: FilterPage(
                   title: S.of(context).labelRelevance,
                   buttonText: S.of(context).buttonSet,
-                  info: S.of(context).infoRelevance,
+                  info: S.of(context).infoRelevanceNothingSelected +
```

Comment:
May be we should make one text for both

Reply:
I prefer not having very long strings defined. Plus, maybe in the future our permissions will only allow people to select a subset of nodes - if they can't set the relevance to "Anyone", then `infoRelevanceNothingSelected` isn't relevant for them